packages/hash/api/src/model/comment.model.ts

```diff
@@ -0,0 +1,212 @@
+import { TextToken } from "@hashintel/hash-shared/graphql/types";
+import {
+  Entity,
+  EntityConstructorArgs,
+  EntityExternalResolvers,
+  Comment,
+  UnresolvedGQLEntityType,
+  User,
+  Block,
+} from ".";
+import { DbClient } from "../db";
+import {
+  DbCommentProperties,
+  DbTextProperties,
+  EntityType,
+} from "../db/adapter";
+import {
+  SystemTypeName,
+  Comment as GQLComment,
+  LinkedEntityDefinition,
+} from "../graphql/apiTypes.gen";
+
+export type CommentExternalResolvers = EntityExternalResolvers | "contents"; // contents resolved in `src/graphql/resolvers/Comments/linkedEntities.ts`
```

Comment:
I'm keeping this simply because this is the pattern seen in other models (see block.model.ts or page.model.ts, for example), even though I don't see this being an import that is or will be used.